src/comments.ts

```diff
@@ -13,39 +13,39 @@ export const HadError = (user: string | undefined, error: string) => ({
     tag: "had-error",
     status: `${user ? `@${user} — ` : ""}There was an error that prevented me from properly processing this PR:
 
-    ${error}`
+    ${error}`,
```

Comment:
I'm not too excited about these, since it's only `tag`/`status` and always that, so it looks weird.

Reply:
Reverted :heavy_check_mark: